### PR TITLE
Adds colours to output and template cloning capabilities.

### PR DESCRIPTION
**Pull request by saricden**

### CODE REVIEW

**Comment by saricden** [comment]
Fixing the conflict right now, standby. @hpx7

**Comment by saricden** [comment]
Done. @hpx7

File: index.js [hunk 1]
```diff
@@ -1,162 +1,139 @@
...
+    shell.exit(1);
+  }
+
+  // Get user's project name
   projectName = await prompts({
     type: 'text',
     name: 'value',
     message: 'New project name?',
-    validate: value => !value.match(/^[a-z]+(-[a-z]+)*$/) ? `Lowercase letters interspaced with dashes only.` : true
+    // validate: value => !value.match(/^[a-z]+(-[a-z]+)*$/) ? `Lowercase letters interspaced with dashes only.` : true
```
Comment (hpx7) [inline index.js]:
```suggestion
```

**Comment by saricden** [comment]
@hpx7 sorry I missed your review from before to remove the commented out validation pattern. This is resolved now, can we merge this PR? 🍻

**Pull request merged**